tests/test_wave_01.py

```diff
@@ -60,13 +60,14 @@ def test_get_task_not_found(client):
     # Assert
     assert response.status_code == 404
 
-    raise Exception("Complete test with assertion about response body")
+    # raise Exception("Complete test with assertion about response body")
+    
     # *****************************************************************
     # **Complete test with assertion about response body***************
     # *****************************************************************
+    assert response_body == {"message" : "Task 1 is not found"}
```

Comment:
This test fails when I run it, but not due to your implementation of the GET method for /tasks/<task_id>

Here's the output: 
```
E  {'message': 'Task 1 not found'} != {'message': 'Task 1 is not found'}
```

How would you update your assertion so the test passes?